README.md

```diff
@@ -3,13 +3,37 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  - asymptotic analysis is used to compare growth rates for algorithms. This means that constant factors and lower order factors are ignored. a function that has a practical runtime of 2n^2 + 5 would have a much better performance on smaller data sets than a fuction with a runtime of 4242n + 7log(n) + 5555, even if their asymptotic complexities would be evaluated to O(n^2) and O(n) respectively.
+ 
+  - asymptotic analysis operates under assumptions about the data set used, ie best case, worst case, average case. In reality data used may adhere to certain patterns, such as always being partially or nearly sorted. algorithms that have good best case but bad average or worst case complexity may be faster on said data sets, while an algorithm with the same best worst and average case may be less effective.
```

Comment:
In most cases yes, but knowing what the best case, worst case, and average case runtimes for a program are can give us useful insight into how it can be effectively used and a good metric to compare to other algorithms

Reply:
I don't see how this makes asymptotic analysis misleading though.

Reply:
Asymptotic analysis creates a hierarchy based on these assumptions, which can be misleading when applied practical purposes. A full understanding of asymptotic analysis could eliminate some of its misleading nature, but its presentation can be misleading for the reasons specified.